Issue 1923 add security protocol config to docker quickstart

gschmutz opened the pull request:
# Linked issue

Resolves: #1923 

# What kind of change does this PR introduce?

- [ ] Bug fix
- [X] New feature
- [ ] Refactor
- [ ] Docs update
- [ ] CI update

# What is the current behavior?

_In a docker quickstart setup, the Security Protocol is hardcoded to `PLAINTEXT`._

# What is the new behavior?

_a new setting has been added to `application.properties` allowing to overwrite the Security Protocol._

# Requirements (all must be checked before review)

- [X] The pull request title follows [our guidelines](https://github.com/Aiven-Open/klaw/blob/main/CONTRIBUTING.md#guideline-commit-messages)
- [ ] Tests for the changes have been added (if relevant)
- [X] The latest changes from the `main` branch have been pulled
- [ ] `pnpm lint` has been run successfully

aindriu-aiven reviewed (approved):
This looks good to me,

to override the property you can use the following to the cluster api, to get this change we will need to release a new nightly to and update docs to point to this. in this update we can also override the https setting that was introduced in 2.6.0

-e KLAW_DOCKER_KAFKA_CLUSTER_SECURITYPROTOCOL="SSL"

gszecsenyi commented:
Great work, Guido!

aindriu-aiven commented:
@gschmutz Just wanted to say thanks again and I have updated the quickstart guide and landing page in https://www.klaw-project.io/ to point to the 'nightly' docker build so it will now pull in your change as well as resolve the additional error you noted from the https url setting.

gschmutz commented:
> Great work, Guido!

Thanks Gergeley, wasn't much .... :-)

gschmutz commented:
> @gschmutz Just wanted to say thanks again and I have updated the quickstart guide and landing page in https://www.klaw-project.io/ to point to the 'nightly' docker build so it will now pull in your change as well as resolve the additional error you noted from the https url setting.

appreciate the nightly build, just switched to it and it works fine :-)
